config/initializers/omniauth.rb

```diff
@@ -2,7 +2,8 @@
 
 Rails.application.config.middleware.use OmniAuth::Builder do
   provider :facebook, ENV['FACEBOOK_APP_ID'], ENV['FACEBOOK_SECRET'],
-           :scope => 'email', :info_fields => 'name, email'
+  :scope => 'email', :info_fields => 'name, email'
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.